app/controllers/movements/MovementSummaryController.scala

```diff
@@ -30,7 +30,7 @@ import views.html.summary.{arrival_summary_page, departure_summary_page}
 
 import scala.concurrent.ExecutionContext
 
-class SummaryController @Inject()(
+class MovementSummaryController @Inject()(
```

Comment:
Is it worth to change the name to contain `Movement`?

Reply:
huh?

Reply:
`SummaryController` IMO was enough. This whole internal service is about movements, so this `Movement` in the name is redundant.